pkg/sql/exec/utils_test.go

```diff
@@ -372,7 +460,7 @@ func (s *opTestInput) Next(context.Context) coldata.Batch {
 			// If useSel is false, then the selection vector will contain
 			// [0, ..., batchSize] in ascending order.
 			outputIdx := s.selection[j]
-			if tups[j][i] == nil {
+			if tups[j][i] == nil || s.injectNulls {
```

Comment:
This seems to be all-nulls injection. I think we should try injecting random nulls too, although I guess predicting output differences won't be straightforward. I guess at the least we could test that nothing errors or panics.